README.md

```diff
@@ -1,5 +1,8 @@
 GDAL - Geospatial Data Abstraction Library
 ====
+This fork of GDAL is used by the ArcGIS runtime SDKs product line exclusively. The ArcGIS Pro product line uses a custom
+internal version of GDAL for each release. For more information on ArcGIS Pro's gdal, please find the appropriate
```

Comment:
I'm not sure I'd say it uses a custom internal version.  I'd probably say less than this... something like:

```
This fork of GDAL is used by the ArcGIS runtime SDKs product line exclusively, and is independent of the ArcGIS Pro product.  For more information on ArcGIS Pro's gdal, see the documentation at <$URL>
```